documents/processes/review/EXAMPLE.md

```diff
@@ -0,0 +1,38 @@
+# AsyncAPI Review Document
+
+|Project name|Reviewers|Company|
+|------------|---------|-------|
+|Awesome project|Jack Johnson, John Jackson|ACME|
```

Comment:
Should we capture the version of the spec. reviewed (i.e. if there are more than one RC / for future versions)?

Reply:
Added. Thanks for the feedback!